docs/content/latest/deploy/kubernetes/clients.md

```diff
@@ -109,3 +109,137 @@ Status:
         router: no errors
         loadbalancer emulator: no errors
 ```
+
+## Connecting TLS Secured YugabyteDB cluster deployed by Helm Charts
+
+### Within Kubernetes cluster
+
+Copy the following `yb-client.yaml` and use this `kubectl create -f yb-client.yaml` command to create a pod with auto-mounted client certificates.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: yb-client
+  namespace: yb-demo
+spec:
+  containers:
+  - name: yb-client
+    image: yugabytedb/yugabyte-client:latest
+    env:
+    - name: SSL_CERTFILE
+      value: "/root/.yugabytedb/root.crt"
+    volumeMounts:
+    - name: yugabyte-tls-client-cert
+      mountPath: "/root/.yugabytedb/"
+  volumes:
+  - name: yugabyte-tls-client-cert
+    secret:
+      secretName: yugabyte-tls-client-cert
+      defaultMode: 256
+```
+
+Here is an example of a client that uses the `YSQL shell` ([`ysqlsh`](../../../admin/ysqlsh)) to connect.
+
+Use the following command to verify the connection.
+
+```sh
```

Comment:
We should combine these two command examples into a single command example which connects via ysqlsh and shows the result of running `\conninfo` within that shell.

Reply:
Sure, Will do